### PR TITLE
Executive order full listing, teaser, and node view

### PR DESCRIPTION
**Pull request by jordan-caldwell**
## Summary
This PR provides the theming for the executive order full listing, teaser, and node view.

Also included is some some contrib module changes to support the vaccination site as requested by Jason.

## Metadata
| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes/
| Did you perform a self review first? | yes
| `CHANGELOG` reflects changes? | yes
| Did you perform browser testing? | yes
| Risk level | Low
| Relevant links | https://thinkoomph.jira.com/browse/RIGA-47

### CODE REVIEW

**Comment by JMStewart00** [comment]
> @JMStewart00 Can you take this over and address any feedback @ben-hamelin provides? Then merge once approved.

You got it @jordan-caldwell

File: ecms_base/themes/custom/ecms/templates/content/node--executive-order--full.html.twig [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ *
+ * @todo Remove the id attribute (or make it a class), because if that gets
+ *   rendered twice on a page this is invalid CSS for example: two lists
+ *   in different view modes.
+ */
+#}
+
+{% if langcode is null %}
+  {% set langcode = 'en' %}
+{% endif %}
```
Comment (ben-hamelin) [inline ecms_base/themes/custom/ecms/templates/content/node--executive-order--full.html.twig:76]: For langcode to be defined we need to add the following to the ecms.theme file:
`/**
 * Implements hook_preprocess_node().
 */
function ecms_preprocess_node__executive_order(array &$variables): void {

  // We need to know the current language for some requests.
  $language = \Drupal::languageManager()->getCurrentLanguage()->getId();
  $variables['langcode'] = $language;
}`

**Review by ben-hamelin — CHANGES_REQUESTED** [review]
@JMStewart00 Overall this looks good, just need to add the preprocess function to the .theme

**Review by ben-hamelin — APPROVED** [review]
Looks good!
/approved

**Pull request merged**